### PR TITLE
[ENH] Percent signal change from ROIs in single commit

### PR DESCRIPTION
**Pull request by Gilles86**
Hi,

This is a rework of an earlier [pull request](https://github.com/nilearn/nilearn/pull/1688) squashed in one (plus one) commit, that should be much easier to review. Please let me know what you think!

Cheers,
Gilles

### CODE REVIEW

**Comment by kchawla-pi** [comment]
Wow, great work man, I will look at this soon. This was a lot of (re)work for you and for that I am sorry!
Thanks for sticking with it!

PS: I think you meant "Percent signal change in single commit" in the PR title. :~)

**Comment by Gilles86** [comment]
> Wow, great work man, I will look at this soon. This was a lot of (re)work for you and for that I am sorry!
> Thanks for sticking with it!

No worries. After I found out about `git merge --squash ...` it was actually surprisingly little work. :)

> 
> PS: I think you meant "Percent signal change in single commit" in the PR title. :~)
Good one ;)

**Comment by kchawla-pi** [comment]
> No worries. After I found out about git merge --squash ... it was actually surprisingly little work. :)

Teach me  what you did! :~)

File: doc/whats_new.rst [hunk 1]
```diff
@@ -10,6 +10,9 @@ NEW
 - NiftiLabelsMasker now consumes less memory when extracting the signal from a 3D/4D
   image. This is especially noteworthy when extracting signals from large 4D images.
 - New function :func:`nilearn.datasets.fetch_atlas_schaefer_2018`
+- Rework of the standardize-options of :func:`nilearn.signal.clean` and the various Maskers
+  in `nilearn.input_data`. You can now set `standardize` to `zscore` or `psc`. `psc` stands
+  for `Percent Signal Change`, which can be a meaningful metric for BOLD.
```
Comment (GaelVaroquaux) [inline doc/whats_new.rst]: That should now be moved in the section for 0.6. Sorry :$

File: nilearn/signal.py [hunk 2]
```diff
@@ -374,10 +398,6 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
 
     Filtering is only meaningful on evenly-sampled signals.
 
-    According to Lindquist et al. (2018), removal of confounds will be done
-    orthogonally to temporal filters (low- and/or high-pass filters), if both
-    are specified.
-
```
Comment (GaelVaroquaux) [inline nilearn/signal.py]: Why was this removed?

File: nilearn/signal.py [hunk 3]
```diff
@@ -399,8 +419,7 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
         signal, as if all were in the same array.
 
     t_r: float
-        Repetition time, in second (sampling period). Set to None if not
-        specified. Mandatory if used together with low_pass or high_pass.
+        Repetition time, in second (sampling period).
```
Comment (GaelVaroquaux) [inline nilearn/signal.py]: Same thing here: why the change?

File: nilearn/signal.py [hunk 4]
```diff
@@ -430,11 +454,6 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
     Linear Approach". Human Brain Mapping 2, no 4 (1994): 189-210.
     <http://dx.doi.org/10.1002/hbm.460020402>`_
 
-    Orthogonalization between temporal filters and confound removal is based on
-    suggestions in `Lindquist, M., Geuter, S., Wager, T., & Caffo, B. (2018).
-    Modular preprocessing pipelines can reintroduce artifacts into fMRI data.
-    bioRxiv, 407676. <http://dx.doi.org/10.1101/407676>`_
```
Comment (GaelVaroquaux) [inline nilearn/signal.py]: Why was this removed?

**Comment by GaelVaroquaux** [comment]
Overall this looks good, but I commented on a few changes that I don't understand.

**Comment by kchawla-pi** [comment]
Hey @Gilles86 What say we wrap this one up before we get merge conflicts bigger than whats_new? :smile:

**Comment by kchawla-pi** [comment]
@bthirion @jeromedockes It seems Gilles seem does not have resources to allocate to this PR. He has done all the heavy lifting & seemingly the only block is the change in some documentation that he made that Gael would like clarification on.
I propose we review the change & if it doesn't make sense to us, we will exclude the doc changes in question & merge the rest. It can always be added later if necessary.

File: nilearn/signal.py (hunk 2 above)
Comment (bthirion) [reply nilearn/signal.py]: @kchawla-pi Could you restaure this comment ?

File: nilearn/signal.py (hunk 3 above)
Comment (bthirion) [reply nilearn/signal.py]: Same thing here

File: nilearn/signal.py (hunk 4 above)
Comment (bthirion) [reply nilearn/signal.py]: Smae thing here.

**Comment by kchawla-pi** [comment]
Restore the comment? Yes that I can do.

**Comment by kchawla-pi** [comment]
I will move forward today, on the plan outlined above. There's a problem with CircleCI on their side. Once that has been attended to I will merge this.

**Comment by Gilles86** [comment]
Hey guys,
Thank you so much for the work. I was very busy finishing projects and moving to another country/university the past months.

It seems that some changes to the documentation were just sloppy errors, but you have it under control now.

Let me know if I can still help in any way.

Stoked that my work will be included into this amazing package so other people can use it! :)

**Comment by kchawla-pi** [comment]
Thank you for all the contributions @Gilles86 ! We are stoked to include them!
Congratulations on the new life chapter!

**Comment by kchawla-pi** [comment]
Once the CircleCI difficulty has been attended to we can resume the merge train.
